source/documentation/06-api-reference.md

```diff
@@ -187,8 +188,10 @@ These are the possible values of the card_brand parameter.
 
 ## API rate limits
 
-There is a maximum rate limit for requests to the API from your service account. The limit is high and most services are unlikely ever to exceed it.
+We rate limit POST requests to the API at 15 requests per second per government service. 
 
-If you do exceed the limit (that is, send a large number of requests in a short amount of time), you will receive P0900 errors. If this happens, you can attempt any rate-limited requests again after a second has passed.
+We also rate limit GET requests but currently at a very high level. We will publish an official limit for GET API requests in the next few weeks.
 
-Please contact us if you want to discuss the rate limiting applied to your service account.
+If you make more requests than the limit, we will return HTTP status code "429" (Too Many Requests) and error code P0900. If you hit our rate limits, you can retry your attempts after a second in a reasonable way (i.e., exponential backoff) 
+
+Please contact us if these rate limits cause you any issues or if you have feedback for our approach
```

Comment:
Minor stylistic edit and suggestion to add an anchor to the support section:

> Please [contact us](#support-contact-and-more-information) if you would like to discuss rate limiting applied to your service account, or give us feedback.

Reply:
Updated as per recommendation